packages/api/cms-api/package.json

```diff
@@ -37,11 +37,10 @@
         "@azure/openai": "1.0.0-beta.11",
         "@azure/storage-blob": "^12.23.0",
         "@fast-csv/parse": "^4.3.6",
-        "@golevelup/nestjs-discovery": "^3.0.0",
+        "@golevelup/nestjs-discovery": "^4.0.2",
         "@hapi/accept": "^5.0.2",
-        "@nestjs/jwt": "^9.0.0",
```

Comment:
@fraxachun I've removed this dependency since it seems that it isn't needed any longer. Is this okay?

Reply:
Added it again: 4f3d6a02318a89f4fa31e88308283a1957cefdc0